spec/features/admin_can_bulk_deactivate_suppliers_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin can bulk deactivate suppliers' do
+  let!(:supplier) do
+    supplier = create(:supplier,
+                      name: 'Aardvark (UK) Ltd',
+                      coda_reference: 'C099999',
+                      salesforce_id: '001b000003FAKEFAKE')
+    supplier
+  end
+
+  let!(:agreement) do
+    supplier.agreements.create!(framework: fm1234)
+  end
+
+  let(:fm1234) do
+    create(:framework, short_name: 'FM1234')
+  end
+
+  before do
+    sign_in_as_admin
+  end
+
+  context 'with a valid CSV' do
+    scenario 'deactivates suppliers on specified frameworks' do
```

Comment:
How about assigning this supplier to an additional framework/agreement, so that we have confidence that the deactivation is happening on the framework specified in the CSV?

Reply:
Have done this and added some more specific checks to the test.